nodes/persistence-node/src/services/CacheRunner.ts

```diff
@@ -171,18 +172,24 @@ export class CacheRunner {
         const ipfsHash = getIpfsHashFromContenthash(contenthash);
   
         this.deps.logger.log("Retrieved IPFS hash for ENS domain");
-        const newlyPinned = await this.processEnsIpfs(ensNode, ipfsHash);
+        const status = await this.processEnsIpfs(ensNode, ipfsHash);
 
-        if(newlyPinned) {
+        if (status === ProcessEnsIpfsStatus.Pinned) {
           pinnedCnt++;
         }
+        if (status === ProcessEnsIpfsStatus.Error) {
+          unresponsiveCnt++;
+        }
       } catch(ex) {
+        this.deps.storage.unresponsiveEnsNodes[ensNode] = true;
```

Comment:
According to message in `logger.log` below, we expect failed content hash retrieval to result in node being added to unresponsive, but that would never happen if `this.deps.ensPublicResolver.contenthash(ensNode)` threw exception.